components/globals/Alert/Alert.tsx

```diff
@@ -0,0 +1,310 @@
+import { CircleCheckIcon, InfoIcon, WarningIcon } from "@components/form-builder/icons";
+import React, { ReactNode, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+export enum ErrorStatus {
+  SUCCESS = "SUCCESS",
+  WARNING = "WARNING",
+  ERROR = "ERROR",
+  INFO = "INFO",
+}
+
+export const Title = ({
+  children,
+  level = "h2",
+  status,
+}: {
+  children: string;
+  level?: "h2" | "h3" | "h4";
+  status?: ErrorStatus;
+}) => {
+  if (typeof children !== "string") {
+    return <>{children}</>;
+  }
+  const Wrapper = `${level}` as keyof JSX.IntrinsicElements;
+  const className = status ? `${defaultClasses.text[status]}` : "";
+
+  return <Wrapper className={`${className}`}>{children}</Wrapper>;
+};
+
+export const Body = ({ children }: { children: JSX.Element | string }) => {
+  return <>{children}</>;
```

Comment:
Do we need the React fragment here?  I don't see a case where we could pass children without 

```
<Body>
            <>
              {alertBody && <>{alertBody}</>}
              {content}
            </>
          </Body>
```